assets/src/common/components/pre-publish-panel.js

```diff
@@ -19,9 +19,10 @@ import { validateFeaturedImage } from '../helpers';
 /**
  * Conditionally adds a notice to the pre-publish panel for the featured image.
  *
- * @param {Object}  featuredMedia Media object.
- * @param {Array}   dimensions    Required image dimensions.
- * @param {boolean} required      Whether selecting a featured image is required.
+ * @param {Object} props Component props.
+ * @param {Object} props.featuredMedia Media object.
+ * @param {Array} props.dimensions Required image dimensions.
```

Comment:
Indentation alignment for the parameter name is lost here. It's not important to me in any way, but I don't know if it is according to some coding standard / style guide.

Reply:
Similarly in a few other places:

https://github.com/ampproject/amp-wp/pull/3150/files#diff-772941062b46bc04ec5e6d20e2dcd9afR15-R19
https://github.com/ampproject/amp-wp/pull/3150/files#diff-ed917cb11654e9f341941fb5ed4e2361R16-R18

Reply:
Yeah we are not really consistent with that 😞 
The thing is, there is no ESLint rule that checks for alignment here and could enforce either style. So if you see some alignment, it has probably been added manually in the past.

I think we should move away from this vertical alignment anyway, because - other than in the PHP tooling - it cannot be enforced. Also, type definitions can be quite long for complex objects, making it hard to read with this alignment.